<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EndCalc</class>
 <widget class="QDialog" name="EndCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>End</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QLineEdit" name="paste_a_lineEdit">
        <property name="toolTipDuration">
         <number>1</number>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string>alfa</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>alfa</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="paste_L0_lineEdit">
        <property name="whatsThis">
         <string>L0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>L0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="pastePushButton">
        <property name="text">
         <string>Paste</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_mode">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton_degree">
        <property name="text">
         <string>degree ( comp/exp ) </string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_work">
        <property name="text">
         <string>work</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QGroupBox" name="groupBox_mix">
     <property name="title">
      <string>Mix</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="le_a">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>Alfa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_L0">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>L0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_capAir">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Air capacity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_capFuel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Fuel capacity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_calcMix">
        <property name="text">
         <string>Calc</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="output_mix">
        <property name="text">
         <string>Mix capacity</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_air">
     <property name="title">
      <string>Air</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="le_tempAir">
        <property name="placeholderText">
         <string>temp</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_degreeAir">
        <property name="placeholderText">
         <string>comp degree</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_workAir">
        <property name="placeholderText">
         <string>work</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_effAir">
        <property name="placeholderText">
         <string>eff</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_calcAir">
        <property name="text">
         <string>Calc</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="output_air">
        <property name="text">
         <string>Air capacity</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_fuel">
     <property name="title">
      <string>Fuel</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLineEdit" name="le_tempFuel">
        <property name="placeholderText">
         <string>temp</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_degreeFuel">
        <property name="placeholderText">
         <string>exp degree</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_workFuel">
        <property name="placeholderText">
         <string>work</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_effFuel">
        <property name="placeholderText">
         <string>eff</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_calcFuel">
        <property name="text">
         <string>Calc</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="output_fuel">
        <property name="text">
         <string>Fuel capacity</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
